docs/user-guide.md

```diff
@@ -38,7 +38,20 @@ This guide provides detailed instructions on how to configure your Liberty proje
 
 ### Settings
 
-To ensure that the Liberty Config Language Server starts properly, define the `xml.java.home` property in your VS Code settings or set the system `JAVA_HOME` environment variable to point to Java 17 .
+Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server) and the [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta) start properly. A toast message alerts you if the Liberty Language Config server fails to run.
```

Comment:
Use Liberty Config Language Server in last sentence too.

Also, is there a toast message if LSP4Jakarta fails to run?

Reply:
Ah. This is the clarification.
The check in Liberty Tools for VS Code is for both LSP4Jakarta and LCLS. The compliant JDK is used to start both servers. So instead, maybe the toast should say Liberty Tools for VS Code requires Java 17.
We do not have an alert from our plugin if Lemminx fails for LCLS. That is an entirely separate issue.

Reply:
So maybe the last sentence should be: "A toast message alerts you if the either of the language servers fail to run."

And then we should open an issue to update the toast message